src/future/server/mod.rs

```diff
@@ -325,15 +331,15 @@ struct BindStream<S, St> {
 
 impl<S, St> fmt::Debug for BindStream<S, St>
     where S: fmt::Debug,
-          St: fmt::Debug,
+          St: fmt::Debug
 {
     fn fmt(&self, f: &mut fmt::Formatter) -> Result<(), fmt::Error> {
         const HANDLE: &'static &'static str = &"Handle { .. }";
         f.debug_struct("BindStream")
```

Comment:
I would have expected `.debug_struct` to be on the next line, to be consistent with what I see elsewhere.